src/estimators/advi.jl

```diff
@@ -0,0 +1,57 @@
+
+struct ADVI{Tlogπ, B <: Union{Function, Bijectors.Inverse{<:Bijectors.Bijector}}} <: AbstractGradientEstimator
```

Comment:
```suggestion
struct ADVI{Tlogπ,B} <: AbstractGradientEstimator
```
IMO no need to restrict the type here.

In particular, restricting to `Bijectors.Inverse` is not the correct thing to do as `inverse(f)` does not always return `Bijectors.Inverse` (in fact, `inverse` is from InverseFunctions.jl).

Reply:
Yes, I'm not entirely sure how to deal with bijectors yet. If reconstructing the distribution as a `TransformedDistribution` turns out to be efficient, (depending on how we decide to flatten-deflatten the parameters) the ADVI object simply won't need to contain a bijector.